dynomitemanager-core/src/main/java/com/netflix/dynomitemanager/resources/DynomiteAdmin.java

```diff
@@ -157,11 +159,30 @@ public Response getSeeds() {
     @Path("/{cluster_describe : (?i)cluster_describe}")
     public Response getClusterDescribe() {
         try {
-            final List<String> nodes = ii.getClusterInfo();
+            List<String> nodes = new LinkedList<String>();
+
+            for (AppsInstance ins : ii.getClusterInfo()) {
+                logger.debug("Adding node: " + ins.getInstanceId());
+                if (config.getDynomiteHashtag().isEmpty()) {
+                    nodes.add("{" + "\"token\":" + "\"" + ins.getToken() + "\"," + "\"hostname\":" + "\""
+                            + ins.getHostName() + "\"," + "\"rack\":" + "\"" + ins.getRack() + "\"," + "\"ip\":" + "\""
+                            + ins.getHostIP() + "\"," + "\"zone\":" + "\"" + ins.getZone() + "\"," + "\"dc\":" + "\""
+                            + ins.getDatacenter() + "\"" + "}");
+                } else {
+                    nodes.add("{" + "\"token\":" + "\"" + ins.getToken() + "\"," + "\"hostname\":" + "\""
+                            + ins.getHostName() + "\"," + "\"rack\":" + "\"" + ins.getRack() + "\"," + "\"ip\":" + "\""
+                            + ins.getHostIP() + "\"," + "\"zone\":" + "\"" + ins.getZone() + "\"," + "\"dc\":" + "\""
+                            + ins.getDatacenter() + "\"" + "\"hashtag\":" + "\"" + config.getDynomiteHashtag() + "\""
+                            + "}");
```

Comment:
combine the code and add hashtag later 

if !config.getDynomiteHashtag().isEmpty()

Reply:
Completed!